### PR TITLE
Updating Spanish translation of the Web UI

### PR DESCRIPTION
**Pull request by n4al**
Updated messages.po to match the current state of the code.

### CODE REVIEW

File: python/web/src/translations/es/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -1,111 +1,112 @@
...
 msgid ""
 "Cannot insert an image for %(device_type)s into a %(current_device_type)s device"
 msgstr ""
 "No se puede insertar una imagen para %(device_type)s en un dispositivo "
 "%(current_device_type)s"
 
-#: socket_cmds.py:39
+#: src/return_code_mapper.py:44
+msgid "return_code"
+msgstr "return_code"
```
Comment (rdmark) [inline python/web/src/translations/es/LC_MESSAGES/messages.po:107]: This is not correct. What command did you use to generate the pot file? It shouldn't pull in the "lazy" placeholder strings.

Comment (n4al) [reply python/web/src/translations/es/LC_MESSAGES/messages.po:107]: I used this one:
`pybabel extract -F babel.cfg -k lazy_gettext -o messages.pot .`

the directory was RASCSI/python/web but I had to update babel.cfg to ignore venv

Comment (rdmark) [reply python/web/src/translations/es/LC_MESSAGES/messages.po:107]: Weird, why did you need to ignore venv? The pybabel tool should be extracting the relevant strings regardless.

Comment (rdmark) [reply python/web/src/translations/es/LC_MESSAGES/messages.po:107]: Anyways, you should not include the lazy_gettext part!

Comment (n4al) [reply python/web/src/translations/es/LC_MESSAGES/messages.po]: Updated it.

**Review by rdmark — APPROVED** [review]

**Pull request merged**